From 8606b0f418c08aaba7f2565b005f9c00f8681ca1 Mon Sep 17 00:00:00 2001
From: David Seifert <soap@gentoo.org>
Date: Fri, 13 Jan 2017 16:14:01 +0100
Subject: [PATCH 4/4] When using libtool, use LTLIBICONV instead.

* This is required, as otherwise -Wl,--as-needed could fail.
---
 src/flac/Makefile.am     | 2 +-
 src/metaflac/Makefile.am | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/flac/Makefile.am b/src/flac/Makefile.am
index bf3bf468..fe6fa489 100644
--- a/src/flac/Makefile.am
+++ b/src/flac/Makefile.am
@@ -60,7 +60,7 @@ flac_LDADD = \
 	$(top_builddir)/src/share/replaygain_synthesis/libreplaygain_synthesis.la \
 	$(top_builddir)/src/libFLAC/libFLAC.la \
 	$(win_utf8_lib) \
-	@LIBICONV@ \
+	@LTLIBICONV@ \
 	-lm
 
 CLEANFILES = flac.exe
diff --git a/src/metaflac/Makefile.am b/src/metaflac/Makefile.am
index 7a4ec6b6..ec201ddf 100644
--- a/src/metaflac/Makefile.am
+++ b/src/metaflac/Makefile.am
@@ -55,6 +55,6 @@ metaflac_LDADD = \
 	$(top_builddir)/src/share/utf8/libutf8.la \
 	$(top_builddir)/src/libFLAC/libFLAC.la \
 	$(win_utf8_lib) \
-	@LIBICONV@
+	@LTLIBICONV@
 
 CLEANFILES = metaflac.exe
-- 
2.11.0

